scripts/aws/cfn/microservice.yml

```diff
@@ -493,6 +493,7 @@ Resources:
           AUDIENCE: !Ref TokenExchangeLambdaEnvironmentAudience
           SSM_PARAMETER_STORE_TIMEOUT_MILLIS: 5000
```

Comment:
Questo è il tempo di cache delle chiavi? possiamo aumentarlo anche di un ordine di grandezza.

Reply:
@MTurraPpa no, questo è il timeout della richiesta http verso il layer che ha i parametri in cache. La cache per il valore è già impostata al massimo di default, ossia 300 secondi.